fix/anko.go

```diff
@@ -12,30 +14,36 @@ func Anko(content string) (string, error) {
 	bb := &bytes.Buffer{}
 
 	lines := strings.Split(content, "\n")
+	l := len(lines)
+	re := regexp.MustCompile(`,\s*func\(t\)\s*{`)
 
-	// fix create_table
-	inCreateTable := false
-	for i := 0; i < len(lines); i++ {
+	for i := 0; i < l; i++ {
 		line := lines[i]
 		tl := strings.TrimSpace(line)
 		if strings.HasPrefix(tl, "create_table") {
-			line = strings.Replace(line, ", func(t) {", ") {", -1)
-			inCreateTable = true
-		}
-		if strings.HasPrefix(tl, "}") && inCreateTable {
-			inCreateTable = false
-		}
-		if strings.HasPrefix(tl, "})") && inCreateTable {
-			line = "}"
-			inCreateTable = false
-		}
-		lines[i] = line
-	}
-
-	// fix raw
-	for i, line := range lines {
-		tl := strings.TrimSpace(line)
-		if strings.HasPrefix(tl, "raw(") {
+			// skip already converted create_table
+			if re.MatchString(line) {
+				// fix create_table
+				line = re.ReplaceAllString(line, ") {")
+				ll := i
+				lines[i] = line
+				for {
+					if strings.HasPrefix(tl, "})") {
+						line = "}"
+						break
+					} else if strings.HasPrefix(tl, "}") {
+						break
+					}
+					i++
+					line = lines[i]
+					tl = strings.TrimSpace(line)
+					if l == i {
```

Comment:
you are checking for overflow after trying to access an item past the length of the array on 38.